### DISCUSSION TITLE
聚合模型使用GROUPING SETS 做聚合计算，返回key不正确

### DISCUSSION THREAD

**Discussion by yangjun616** [apache/doris#6583] [A - General / Q&A]
查询语句：SELECT a, b, bitmap_union_count( c ) FROM tab1 GROUP BY GROUPING SETS ( (a, b), (a), (b) );
返回值中a,b 不会出现null，而是被group by a,b 的值覆盖

**Comment by EmmyMiao87** [comment]
哪个版本的代码？

**Reply by banjin** [reply]
PALO-0.15.1-rc03 版本，也遇到这种问题
